apps/sms/js/recipients.js

```diff
@@ -1,5 +1,12 @@
-/*global GestureDetector, Navigation, SharedComponents, Utils,
-         Settings */
+/*global
```

Comment:
I noticed you started to align globals differently :) I'm fine with any way, but if you think it's better could you please update/extend our guidelines?